### ISSUE TITLE
Latest Windows update, KB5029263, incompatible with Open Shell 

### ISSUE THREAD

**Issue by ivanlan9** [Open-Shell/Open-Shell-Menu#1662]
### Describe the bug

OpenShell custom button replacement does not replace the ugly Windows 11 start button.
![ScreenHunter_198 Aug  10 17 33](https://github.com/Open-Shell/Open-Shell-Menu/assets/7977158/4b8cacfd-afb4-4209-9b2e-93a320bad0c0)

### Area of issue

Start menu, Taskbar

### To reproduce

Install latest Windows 11 Cumulative update with KB5029263 and reboot.  If you are using ExplorerPatcher FileExplorer will crash every ten seconds or more.  If you are using Fences, ExplorerPatcher's crashes will screw that up too.

But at the very least, you will see the ugly Windows 11 start button hiding behind the OpenShell or custom button.

### Expected behavior

I expected my custom button to act like it did before I installed update KB5029263. It did not. There were other apps in even worse shape - ExplorerPatcher can't even run, it crashes every ten seconds.

### Open-Shell version

4.4.190

### Windows version

Windows 11 22H2 (with KB5029263 )

### Additional context

Clearly this is not a problem caused by OpenShell, but by M$.  You'll have to play catch up.  Good luck.

I love OpenShell and hate that M$ screws you over.  You guys do a MUCH better job than they do.

**Comment by ge0rdi** [comment]
> OpenShell custom button replacement does not replace the ugly Windows 11 start button.

Note that on Windows11 we are unable to hide original start button. Its because the taskbar now uses completely different framework (XAML) than in the past.

Our standard `Aero` button should be placed _over_ original start button, thus it should overlap it completely.

But for custom start button images you have to properly align it and make sure it is opaque so that original start button is not visible behind it.

**Comment by ge0rdi** [comment]
I have fully updated Windows11 22H2 (with KB5029263) and I didn't notice any change in behavior comparing to previous updates. Though MS is enabling some features gradually so it may happen that something is just not enabled for me.

Anyways, the _issue_ with custom start button image is there since Windows 11.
You just need to adjust your image so that it is shifted few pixels to the right. I thing any decent image editor should be able to do that easily.

**Comment by ivanlan9** [comment]
That has not been the case until this latest update.  I've been running Windows 11 for three months, and until yesterday, my custom button either hid or replaced the ugly W11 start button.

**Comment by ivanlan9** [comment]
I've used a center aligned taskbar enough to know I dislike it, and there has been no need to shift my custom image to the right at all--I used the identical image on W10 for many years, carried it over to W11 where it's behaved perfectly until yesterday.

**Comment by ge0rdi** [comment]
> where it's behaved perfectly until yesterday

So you were using _left aligned_ taskbar and it worked well and then yesterday it started to misbehave?

That is really weird. According to screenshot you provide the original start button is at correct position (you can see it is slightly shifted to the right). So your custom image has to be also shifted right.

I don't understand how it could have worked for you before :(
Maybe it is some interaction with other customization SW you are using?

I don't see any problem with `Open-Shell` here as it behaves just as it should on Windows 11.

**Comment by ivanlan9** [comment]
So I made an image that shifted my button 40 pixels to the right.  Now I have a 40-pixel space on the left--and the ugly W11 button sticks out exactly the same distance on the right that it did before I changed the image.  See attached.
![ScreenHunter_200 Aug  11 09 24](https://github.com/Open-Shell/Open-Shell-Menu/assets/7977158/57f6d0bd-720c-4fdc-be03-86aac7bfac17)

**Comment by ivanlan9** [comment]
Image size was set to 40, as it had always been.  I have now unshifted the image, aligned to corner of screen (which made my button smaller and the ugly button stick out more), AND button image size set to 48.  The combination works, but it is not as aesthetic as it was up until I did the W11 update yesterday.  The button is now big enough to hide the ugly button, but it is too big. ... With ExplorerPatcher, I could adjust the taskbar button size, but after the same update which messed up this button placement, EP crashes every ten seconds and is unusable.  See attached image again.
![ScreenHunter_201 Aug  11 11 26](https://github.com/Open-Shell/Open-Shell-Menu/assets/7977158/a649c70d-730b-47c5-a29c-84e693d2aea4)

**Comment by ge0rdi** [comment]
Well, I think you just need to wait for fixed version of `ExplorerPatcher`.
It is quite possible that it tweaked the taskbar in a way that original start button was not shifted right (as it is in standard windows).
And thus `Open-Shell` applied the custom image without any issue.

**Comment by ivanlan9** [comment]
I figured it out.  Turns out that I was running Stardock WindowBlinds--with very little customization. What I didn't know is that it hides the ugly start button, thus allowing OpenShell's buttons to be smaller than the ugly one.  It seems to work best when the opacity (on WindowBlinds) is set to 0%, otherwise the patterns for the theme used show through.  See the attached screenshot.

Thanks to everyone for the help!  I'll close this issue.
![ScreenHunter_202 Aug  11 17 50](https://github.com/Open-Shell/Open-Shell-Menu/assets/7977158/458d977d-bfba-4be7-a3c7-b6c1de9cebba)
